### DISCUSSION TITLE
Any extension for "mark previous as read"?

### DISCUSSION THREAD

**Discussion by kmac** [FreshRSS/FreshRSS#6122] [Q&A] [answered]
The one thing I miss from tt-rss is the ability, from the UI, to quickly mark all previous articles as read.  I know I can use the Alt-r shortcut for this, but sometimes a UI button is nice.  

Does anyone know of an extension that adds this?

**Comment by math-GH** [comment, marked as answer]
There is an extension in creation that could fulfill your wish: https://github.com/FreshRSS/Extensions/pull/205
